File: Q6.py
nome = input("Digite o seu nome: ")
número_de_vezes = int(input("Digite a quantidade de vezes que deseja que o seu nome apareça na tela: "))
vezes = 1
for contador in range(0, número_de_vezes):
    print(f"{vezes}ª Vez: {nome}.")
    vezes += 1

File: Q6.java
import java.util.Scanner;

public class Q6
{
    public static Scanner leia = new Scanner(System.in);
    public static void main(String args[])
    {
        String nome;
        int número_de_vezes, vezes = 1;
        System.out.print("Digite o seu nome: ");
        nome = leia.nextLine();
        System.out.print("Digite a quantidade de vezes que deseja que o seu nome apareça na tela: ");
        número_de_vezes = leia.nextInt();
        for (int contador = 0; contador < número_de_vezes; contador ++, vezes ++)
        {
            System.out.printf("%dª Vez: %s.\n", vezes, nome);
        }
    }
}